exercises/hangman/Hints.md

```diff
@@ -0,0 +1,3 @@
+## Hints
+This exercise requires to work with events. For more information, see [this page]
```

Comment:
"This exercise requires to work with events" should be "This exercise requires you to work with events"